src/guide/jbs-jdk-bug-system.md

```diff
@@ -209,7 +209,7 @@ Now that the issue is in the right component and has the basic information, the
     * This may involve reproducing the bug, if doing so is fast and easy.
     * In addition to the version where the bug was found, take special care to also investigate if the bug affects mainline.
       * See [Indicating what releases an issue is applicable to](#indicating-what-releases-an-issue-is-applicable-to) for more details.
-    * For enhancements the [Affects Version]{.jbs-field} should be empty unless you feel that it is only relevant to a particular release family, and should not go into a future mainline release.
+    * For enhancements the [Affects Version]{.jbs-field} should be empty, unless you feel that it is only relevant to a particular release family, and should not go into a future mainline release.
```

Comment:
To me, the comma before “unless” is redundant, conditional clauses after a main clause don't usually have commas before the conjunction that introduces it; however, this conditional clause is very long… Still, I wouldn't put a comma there.

With the comma before “and”, I feel the second conditional needs _a subject_ “it”; and removing “you feel that” makes the sentence shorter and clearer:
```suggestion
    * For enhancements the [Affects Version]{.jbs-field} should be empty, unless it is only relevant to a particular release family, and (unless) it should not go into a future mainline release.
```
To make the sentence even clearer, you may repeat the conjunction: “_unless_ it should…”, I put it in parentheses in the suggestion.

Reply:
agree that the feel part should go as well as the comma